pkgs/os-specific/darwin/apple-source-releases/adv_cmds/package.nix

```diff
@@ -0,0 +1,121 @@
+{
+  lib,
+  apple-sdk,
+  apple-sdk_11,
+  bison,
+  fetchFromGitHub,
+  flex,
+  mkAppleDerivation,
+  ncurses,
+  perl,
+  pkg-config,
+  stdenvNoCC,
+}:
+
+let
+  Libc = apple-sdk_11.sourceRelease "Libc";
+
+  # The headers have moved around between releases. Using one version is easier.
+  libplatform = fetchFromGitHub {
+    owner = "apple-oss-distributions";
+    repo = "libplatform";
+    rev = "libplatform-316.100.10";
+    hash = "sha256-gojt3sWOr7XO2yYI/B1CmNLTPFieSfoNtlOgQahOCok=";
+  };
+
+  # Use the same vendored version of xnu as system_cmds, so it only has to download once.
```

Comment:
It’s not an invariant. It’s just a reason for a lazy copy/paste. Once we’re on a newer SDK by default, it will probably be fine to use `apple-sdk.sourceRelease "xnu"`.

Reply:
I cleaned up the source releases to use `apple-sdk.sourceRelease` where possible. libpcap can switch once the 15.0 SDK with its releases is available in nixpkgs.